src/main/scala/com/tuplejump/kafka/connect/cassandra/CassandraSinkTask.scala

```diff
@@ -16,28 +16,32 @@
 
 package com.tuplejump.kafka.connect.cassandra
 
-import java.util.{Collection => JCollection, Map => JMap, Date => JDate}
+import java.util.{Collection => JCollection, Date => JDate, Map => JMap}
```

Comment:
Since I had few changes in the File, I had organized the imports as per the style we use usually.